baseapp-auth/baseapp_auth/tests/unit/test_emails.py

```diff
@@ -0,0 +1,17 @@
+import baseapp_auth.tests.helpers as h
+import pytest
+
+from ...emails import remove_superuser_notification_email
+
+UserFactory = h.get_user_factory()
+
+pytestmark = pytest.mark.django_db
+
+
+def test_remove_superuser_notification_email(outbox):
+    UserFactory(is_superuser=True)
```

Comment:
This line seems not useful for the tests, but it is actually necessary for having a recipient_list in the e-mail, and then we will get an e-mail detected in `outbox`